src/styles/blocks/productCard/product.scss

```diff
@@ -0,0 +1,113 @@
+.product {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  transition: transform 300ms;
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &__picture {
+    background-image: url(../images/imac.jpeg);
+    background-size: $imageWidth $imageHeight;
+    width: $imageWidth;
+    height: $imageHeight;
+
+    &--modifier {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $mainTextColor;
+
+    &--modifier {
+      margin-bottom: 4px;
+    }
+  }
+
+  &:hover &__name {
+    color: #34568b;
+  }
```

Comment:
Consider adding a transition effect for the color change on hover to make the interaction smoother. You can use `transition: color 300ms;`.